pkg/builder/builder.go

```diff
@@ -126,7 +126,8 @@ func BuildHlSvc(nm types.NamespacedName, vdb *vapi.VerticaDB) *corev1.Service {
 }
 
 // buildConfigVolumeMount returns the volume mount for config.
-// If vclusterops flag is enabled we mount only /opt/vertica/config/https_certs
+// If vclusterops flag is enabled we mount only
+// /opt/vertica/config/node_management_agent.pid
```

Comment:
Is this true? It looks like we don't have any special logic for vclusterops now.

Reply:
Yes, if we don't persist /opt/vertica/config/node_management_agent.pid, nma will hit permission denied.
It is the same reason why we still need to have a volume for/opt/vertica/log